### PR TITLE
Add Backblaze as a blockable service

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
https://github.com/AdguardTeam/HostlistsRegistry/pull/461

### CODE REVIEW

File: services/backblaze.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+id: backblaze
+name: Blackblaze
```
Comment (slavaleleka) [inline services/backblaze.yml:2]: back or black ?

**Review by slavaleleka — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Comment by Alex-302** [comment]
Why this site should be blocked?

**Comment by Sergey-Lyapin** [comment]
@paragm

**Comment by Sergey-Lyapin** [comment]
We decided that there was no point in adding this service because it is used for backup and hosting, an enterprise service.

**Pull request closed without merging**